script/Arcueid/Day 1/01_00_ARC01_6.txt

```diff
@@ -149,7 +149,7 @@ She smiles mischievously.
 [sha:b040c78ce6e8cd6e8751a59a18c9c28f2f069ce2]{
 -- Page 10019, Offset 1929.
 -- 「…………まいったな」
-"......I give up."
+"......Dang."
```

Comment:
Hmm does dang really make sense here? I feel like it's more of a "what have I gotten myself into"

Reply:
got an alternative? i wasnt too hot on dang either but the original felt off too

Reply:
Not sure. "Good grief" is way too Charlie Brown, and the entire "...Just what have I gotten myself into" is maybe kinda long. "I give up" might be tied for least bad, but see if Ben has any ideas

Reply:
not really. I _do not_ stand with "Dang.", but the previous version was ok. Maybe "...this doesn't bode well"

Reply:
back to i give up it is